shell_words/pubspec.yaml

```diff
@@ -0,0 +1,14 @@
+name: shell_words
+description: A sample command-line application.
+# version: 1.0.0
+# homepage: https://www.example.com
+
+environment:
+  sdk: '>=2.7.0 <3.0.0'
+
+dependencies:
+  unicode: ^0.2.2
+
+dev_dependencies:
+  pedantic: ^1.8.0
+  test: ^1.6.0
```

Comment:
Should we upgrade all dependencies to the latest?

pedantic: ^1.8.0 to pedantic: ^1.9.0
pedantic 1.9.0 Enforce 17 new lint rules.

test package has latest version of 1.12.0. I think it has some substantial bug fixes and the only minimum SDK version is 2.4.0 which we are already in.